_source/_posts/2021-10-08-edunham-intro.md

```diff
@@ -0,0 +1,31 @@
+
+{% img blog/edunham-intro/osl-server.jpeg alt:"opening up a server in the OSL datacenter" width:"800" %}{: .center-image }
+
+I got my start in open source and the tech industry as a student software developer and sysadmin at the [OSU Open Source Lab](https://osuosl.org/). The OSL gives students hands-on expertise with skills essential to those careers, and there weren't enough of us to satisfy all the companies who wanted to hire us as interns! To help address this problem, I created the first iteration of our [DevOps Bootcamp](http://devopsbootcamp.osuosl.org/) training program. I was amazed by how good it felt to take information I was lucky enough to have special access to and share it with others who found it useful. When looking at where to steer my career after university, I considered building on my growing resume of talks to pursue evangelism roles, but it wasn't time yet. I wanted to go out and apply my skills at scale, rather than just talking about them. 
+
+{% img blog/edunham-intro/ferrises.jpeg alt:"Ferris plushies and Rust stickers" width:"800" %}{: .center-image }
+
+As ops for the Rust and Servo teams at [Mozilla](https://research.mozilla.org/), I was again able to play the positive-sum game of sharing the most interesting and useful insights I learned from my unique access to fantastic engineers. But as the sysadmin and speaker roads diverged in that forest of mercurial trees, I stuck with the ops route. 
+
+Missing the conference circuit as growing responsibilities meant less time for talks, I started working on a transition toward advocacy in early 2020. Faced with a choice between making the switch then or building on my ops skills for awhile longer, the engineer in me won out. Switching fields, into a field that was frantically reinventing itself at the time, was just too many simultaneous changes to roll out to prod without any staging environment  or CI available.
+
+Okta's developer advocacy team is in the right place at the right time for me. I get to use [my background](https://edunham.net/pages/talks.html) to help useful information flow out of the company to our community, as well as bringing the community's observations and insights back to contribute to internal decision-making. Our product has made my life better as an engineer better whenever I've worked with it, and increasing knowledge about how to do security well 
+
+{% img blog/edunham-intro/travel.jpg alt:"edunham's face" width:"800" %}{: .center-image }
+
+I've been working remotely since 2018, and I put the time that I would otherwise spend commuting into gardening and an assortment of maker projects, which I sometimes mention on [my blog](https://edunham.net/). I'm a volunteer firefighter and emergency medical responder with my local fire department, and I play tuba and sousaphone poorly but enthusiastically on the now-rare occasion that we can get enough people together to make some music.
```

Comment:
```suggestion
I've been working remotely since 2018, and I put the time that I would otherwise spend commuting into gardening and an assortment of maker projects, which I sometimes mention on [my blog](https://edunham.net/). I'm a volunteer firefighter and emergency medical responder with my local fire department, and I play tuba and sousaphone poorly but enthusiastically on the now-rare occasions when we can get enough people together to make some music.
```